### DISCUSSION TITLE
Area Color Based on Value

### DISCUSSION THREAD

**Discussion by Aizaz-Ahmad** [apexcharts/apexcharts.js#2689] [Q&A]
### Hi, ApexChart lovers!!

I have been working on an Area Chart that has negative and positive data. I want to display the area above zero line in Green color and the area below zero line as Red color. I have read through the docs but I couldn't find a way to do so. Can anyone help me to figure that out? The Screenshot of the graph is attached.

Thanks in Advance!

![image](https://user-images.githubusercontent.com/44976514/135799763-72ea2978-eabc-4c59-8c75-e3528a84136a.png)

**Comment by logemann** [comment]
still not possible afaik

**Comment by iamaliybi** [comment]
I have the same issue
